src/main/java/com/aws/greengrass/mqttbridge/MQTTBridge.java

```diff
@@ -108,9 +124,7 @@ protected MQTTBridge(Topics topics, TopicMapping topicMapping, MessageBridge mes
             }
 
             // otherwise, reinstall to completely refresh this plugin
-            if (Arrays.stream(BridgeConfig.ALL_KEYS).anyMatch(child::childOf)) {
-                requestReinstall();
-            }
+            requestReinstall();
```

Comment:
This seems like a nucleus call to reinstall. Can you confirm that we have good logs telling us why this is happening?
To be safe, I would add a clear log signal before this showing the reason for reinstall.

Reply:
Currently there aren't logs for why, will add those 👍